### DISCUSSION TITLE
［灌水］怎么总有人喜欢砸自己饭碗呢？

### DISCUSSION THREAD

**Discussion by fzdqz** [share121/inter-knot#5110] [灌水]
![Screenshot_2024-09-17-21-31-35-406_tv danmaku bili](https://github.com/user-attachments/assets/6ae7991c-d109-4db9-a470-e9046e2d59ab)
各位绳匠怎么看这种现象？

**Comment by JiuXiat** [comment]
盯～！4496！

**Comment by JiuXiat** [comment]
判定完定，节奏向性内容

**Comment by xingyu42** [comment]
没活了可以咬打火机，这些东西本来如此

**Comment by Loxic123** [comment]
可以看看流量差距，现在AC站就是这b样，搞得小up很难生存<img src="https://bbs-static.miyoushe.com/static/2024/09/04/f2ca6cb970baa36ebb1b8950f3894fb2_4590307880523876022.png" alt="玛拉妮-慌张" width="75" >
